dataplanes/vpp/cmd/dataplanetestclient/dataplane-test-client.go

```diff
@@ -169,18 +169,18 @@ func main() {
 			localSource: &common.LocalMechanism{
 				Type: common.LocalMechanismType_KERNEL_INTERFACE,
 				Parameters: map[string]string{
-					nsmutils.NSMkeyNamespace:        srcNamespace,
-					nsmutils.NSMkeyIPv4:             "2.2.2.2",
-					nsmutils.NSMkeyIPv4PrefixLength: "24",
+					nsmvpp.NSMkeyNamespace:        srcNamespace,
+					nsmvpp.NSMkeyIPv4:             "2.2.2.2",
+					nsmvpp.NSMkeyIPv4PrefixLength: "24",
```

Comment:
So... question... with moving these keys from nsmutils to nsmvpp... does that mean any client or other dataplane implementation will have to link nsmvpp to get the key constants?

Reply:
Oh, that's good point, in this case its better to leave them in nsmutils